From ca7860e25d9cb9e69226604d0cbe87521b36a850 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Thu, 1 Sep 2016 13:09:58 +0200
Subject: [PATCH 0466/1345] net: force network driver registration prior to
 pci

commit  a233a4ab5a57fc4c94535fe998c995ac5564616e from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

With probe deferral, which is needed on A7k/A8k due to CP110 clock
driver support, the only way to force proper order of on-board
and external network interfaces registration is to ensure
compilation of drivers/net would be done performed before other
types of interfaces.

Change-Id: Ie0dd6c981832deb9b64c52f71fe6d5c8298f576b
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32496
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/Makefile |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/Makefile b/drivers/Makefile
index cfabd14..a5f3438 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -14,7 +14,7 @@ obj-$(CONFIG_GENERIC_PHY)	+= phy/
 obj-$(CONFIG_PINCTRL)		+= pinctrl/
 obj-$(CONFIG_GPIOLIB)		+= gpio/
 obj-y				+= pwm/
-
+obj-y				+= net/
 obj-$(CONFIG_PCI)		+= pci/
 obj-$(CONFIG_PCI_ENDPOINT)	+= pci/endpoint/
 # PCI dwc controller drivers
@@ -86,7 +86,6 @@ obj-$(CONFIG_MTD)		+= mtd/
 obj-$(CONFIG_SPI)		+= spi/
 obj-$(CONFIG_SPMI)		+= spmi/
 obj-$(CONFIG_HSI)		+= hsi/
-obj-y				+= net/
 obj-$(CONFIG_ATM)		+= atm/
 obj-$(CONFIG_FUSION)		+= message/
 obj-y				+= firewire/
-- 
1.7.9.5

